README.md

```diff
@@ -54,3 +54,190 @@ For a simple, gentler, introduction to Lagom, have a look at the [Hello World gi
+* [reactive-cli](https://github.com/lightbend/reactive-cli) 0.9.0 or later (Verify with `rp version`)
+
+
+### Deployment Workflow
+
+> Note that this is an alternative to the Operations workflow documented below.
+
+The following workflow is well suited for development purposes. It will build Docker images for each subproject, generate Kubernetes
+YAML, and deploy them into your Minikube using `kubectl`. It also installs the [Reactive Sandbox](https://github.com/lightbend/reactive-sandbox)
+in your Minikube, providing the required dependencies for Kafka, Cassandra, and Elasticsearch.
+
+> Want a fresh Minikube? Run `minikube delete` before the steps below.
+
+`minikube start --memory 6000`
+
+`sbt "deploy minikube"`
+
+> Open the URL this command prints in the browser
+
+```bash
+echo "http://$(minikube ip)"
+```
+
+### Operations Workflow
+
+> Note that this is an alternative to the Development workflow documented above.
+
+This workflow matches more closely how an application would be deployed in a production environment. In this method,
+the images are built by SBT. The [reactive-cli](https://github.com/lightbend/reactive-cli) is then
+used to generate YAML for deployment to the Kubernetes cluster. The steps below focus on Minikube, but could be used on
+a real Kubernetes cluster given access to a Docker registry and configured `kubectl`.
+
+> Want a fresh Minikube? Run `minikube delete` before the steps below.
+
+##### Setup Docker engine context to point to Minikube
+
+```bash
+eval $(minikube docker-env)
+```
+
+##### Install Reactive Sandbox (optional; skip if you have your own Cassandra, Kafka, and Elasticsearch clusters) 
+
+The `reactive-sandbox` includes development-grade (i.e. it will lose your data) installations of Cassandra, 
+Elasticsearch, Kafka, and ZooKeeper. It's packaged as a Helm chart for easy installation into your Kubernetes cluster.
+
+```bash
+helm init
+helm repo add lightbend-helm-charts https://lightbend.github.io/helm-charts
+helm update
+```
+
+Verify that Helm is available (this takes a minute or two):
+
+```bash
+kubectl --namespace kube-system get deploy/tiller-deploy
+```
+
+```
+NAME            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+tiller-deploy   1         1         1            1           3m
+```
+
+Install the sandbox.
+
+```bash
+helm install lightbend-helm-charts/reactive-sandbox --name reactive-sandbox
+```
+
+Verify that it is available (this takes a minute or two):
+
+```bash
+kubectl get deploy/reactive-sandbox
+```
+
+```
+NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+reactive-sandbox   1         1         1            1           1m
+```
+
+##### Build Project
+
+```bash
+sbt clean docker:publishLocal
+```
+
+##### View Images
+
+```bash
+docker images
+```
+
+##### Deploy Projects
+
+Finally, you're ready to deploy the services. Be sure to adjust the secret variables and cassanda service address as necessary.
+
+```bash
+# Secret values, be sure to change them
```

Comment:
I had a customer call recently where their deployment was failing because they didn't specify the application secret. This provided a good opportunity to explain the important of this value. I am in agreement with failing fast. I'll change these to `changeme` to support that.

I agree it can be tricky to figure out why it's failing, but I think it's a good learning experience if you're going to be using K8s.